src/rosecommon/include/components/position.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+// This should not be mandatory here. As a user I don't want to include the database connection when I need a position.
```

Comment:
This is because the user is responsible for loading its own data from the database row. I'd like to use reflection to automatically load the components from the database, but for now I don't really know how to do it. Once I have a better idea on how it is done, I'll change it.